src/app/pcasts/dao/discover_dao.py

```diff
@@ -10,13 +13,59 @@ def request_podcast_ml(url):
   except requests.exceptions.ConnectionError:
     raise Exception('Could not connect to podcast-ml')
 
-def get_series_for_topic(topic_id, user_id):
-  response = request_podcast_ml('/api/v1/series/topic/{}'.format(topic_id))
-  return series_dao.get_multiple_series(response['data']['series_ids'], user_id)
+def get_series_for_topic(topic_id, user_id, offset, max_search):
+  if config.ML_ENABLED:
+    response = request_podcast_ml('/api/v1/series/topic/{}'.format(topic_id))
+    return series_dao.get_multiple_series(response['data']['series_ids'], \
+        user_id)
+  else:
+    topic_id = int(topic_id)
+    series = []
+    if topic_id in topic_utils.topic_id_offset:
+      topic_id = topic_utils.translate_topic_id(topic_id)
+      series = Series.query.\
+          filter(((Series.topic_id.op('&')(topic_id))) == topic_id).\
+          order_by(Series.subscribers_count.desc()).\
+          offset(offset).\
+          limit(max_search).\
+          all()
+    if topic_id in topic_utils.subtopic_id_offset:
+      subtopic_id = topic_utils.translate_subtopic_id(topic_id)
+      series = Series.query.\
+          filter((Series.subtopic_id.op('&')(subtopic_id)) == subtopic_id).\
+          order_by(Series.subscribers_count.desc()).\
+          offset(offset).\
+          limit(max_search).\
+          all()
+    return series
```

Comment:
no exception is thrown if the topic_id is invalid, I'm not sure we'd want to handle that situation by returning an empty list without an error